rules/kafka/kafka-rules.yml

```diff
@@ -0,0 +1,129 @@
+
+groups:
+- name: Kafka_Alerts
+  rules:
+    - alert: Offline_Partiton_Count
+      expr: sum(kafka_controller_KafkaController_OfflinePartitionsCount{executor_name="kafka"})by(service_name)  > 0
+      for: 1m
+      labels:
+        severity: critical
+      annotations:
+        summary: 'Kafka {{ $labels.service_name }}: {{ $value }} partitons offline'
+        description: 'After successful leader election, if the leader for partition dies, then the partition moves to the OfflinePartition state. Offline partitions are not available for reading and writing. Restart the brokers, if needed, and check the logs for errors.'
+
+    - alert: Under_Replicated_Partition_Count
+      expr: sum(kafka_server_ReplicaManager_UnderReplicatedPartitions{executor_name="kafka"})by(service_name) > 0
+      for: 1m
+      labels:
+        severity: critical
+      annotations:
+        summary: 'Kafka {{ $labels.service_name }}: {{ $value }} under replicated partitons'
+        description: 'Under-replicated partitions means that one or more replicas are not available. This is usually because a broker is down.  Restart the broker, and check for errors in the logs.'
+
+    - alert: Active_Controller
+      expr: sum(kafka_controller_KafkaController_ActiveControllerCount{executor_name="kafka"})by(service_name) != 1
+      for: 1m
+      labels:
+        severity: critical
+      annotations:
+        summary: 'Kafka {{ $labels.service_name }}: No active controller'
+        description: 'No broker in the cluster is reporting as the active controller in the last 1 minute interval. During steady state there should be only one active controller per cluster.'
+
+    - alert: Unclean_Leader_Election
+      expr: max(kafka_controller_ControllerStats_UncleanLeaderElectionsPerSec_5MinuteRate{executor_name="kafka"})by(service_name) != 0
+      for: 1m
+      labels:
+        severity: critical
+      annotations:
+        summary: 'Kafka {{ $lables.service_name }}: {{ $value }} unclean leader elections'
```

Comment:
typo? (more of these typos below) `$lables.service_name` -> `$labels.service_name`

Reply:
@gracedo fixed the typos :smile: